code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -50,34 +50,34 @@
 		human.client?.prefs.copy_all_to(human, JOB_SQUAD_LEADER, TRUE, TRUE)
 		arm_equipment(human, /datum/equipment_preset/uscm/leader/cryo, mind == null, TRUE)
 		to_chat(human, SPAN_ROLE_HEADER("You are a Squad Leader in the USCM"))
-		to_chat(human, SPAN_ROLE_BODY("You are here to assist in the defence of the [SSmapping.configs[GROUND_MAP].map_name]. Listen to the chain of command."))
+		to_chat(human, SPAN_ROLE_BODY("You are here to assist in the defence of [SSmapping.configs[GROUND_MAP].map_name]. Listen to the chain of command."))
```

Comment:
These `[SSmapping.configs[GROUND_MAP].map_name]` ones didn't actually have a "The the" problem, but in testing I noticed that the sentence didn't make much sense with a 'The' in it.
For example: `""You are here to assist in the defence of the LV-624. Listen to the chain of command."`

So I figured that since it's so close to the actual subject of this PR, it wouldn't hurt to tweak them too.

Reply:
This is fine.